### PR TITLE
Bug 1936108 - Convert "thunderbird_sheriff" to LDAP group.

### PR DESCRIPTION
**Pull request by jfx2006**

### CODE REVIEW

**Review by bhearsum — APPROVED** [review]

**Comment by bhearsum** [comment]
Failures here are unrelated. https://hg.mozilla.org/projects/toolchains either went away by accident or on purpose. We need to figure out which before merging this (the deploy will fail for the same reason if we merge..).

File: grants.yml [hunk 1]
```diff
@@ -2741,14 +2741,8 @@
...
-        # Geoff Lankow (https://bugzilla.mozilla.org/show_bug.cgi?id=1520433)
-        - login-identity:mozilla-auth0/ad|Mozilla-LDAP|geoff
-        # Magnus Melin (https://bugzilla.mozilla.org/show_bug.cgi?id=1605714)
-        - login-identity:mozilla-auth0/ad|Mozilla-LDAP|mkmelin
-        # Patrick Cloke (https://bugzilla.mozilla.org/show_bug.cgi?id=1595942)
-        - login-identity:mozilla-auth0/ad|Mozilla-LDAP|clokep
-        # Ian Neal (https://bugzilla.mozilla.org/show_bug.cgi?id=1595942)
-        - login-identity:mozilla-auth0/ad|Mozilla-LDAP|iann_cvs
+        # Thunderbird sheriffs, Magnus Melin manager - Bug 1936108
+        - mozilla-group:thunderbird_releng
```
Comment (Eijebong) [inline grants.yml:2745]: Am I missing something or should this be `mozilla-group:thunderbird_sheriff`? (title + https://bugzilla.mozilla.org/show_bug.cgi?id=1933561 seem to indicate that)

Comment (bhearsum) [reply grants.yml:2745]: Nope, you're totally right AFAICT! Thanks for catching that.

**Review by bhearsum — CHANGES_REQUESTED** [review]

File: grants.yml (hunk 1 above)
Comment (jfx2006) [reply grants.yml:2745]: 🤦🏼‍♂️

**Review by bhearsum — APPROVED** [review]
lgtm; we will hold off merging now because https://github.com/mozilla-releng/fxci-config/pull/204 has caused issues with the apply that happens on pushes to `main`

**Pull request merged**